docs/uninstallation_guide.md

```diff
@@ -0,0 +1,20 @@
+# Uninstallation Guide
+
+This guide describes how to uninstall Open Service Mesh (OSM) from a Kubernetes cluster using the `osm` CLI.
+
+## Prerequisites
+- Kubernetes cluster with OSM installed
+- The [osm CLI](installation_guide.md#Set-up-the-OSM-CLI)
+
+## Uninstall OSM
+Use the `osm` CLI to uninstall the OSM control plane from a Kubernetes cluster.
```

Comment:
Should we also document how to uninstall using Helm, considering the installation can be done directly via Helm?

Reply:
I can follow up in a different PR so that this script can be merged.

Reply:
See #2197